src/main/java/com/myjeeva/digitalocean/pojo/VolumeAction.java

```diff
@@ -0,0 +1,93 @@
+package com.myjeeva.digitalocean.pojo;
+
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+import com.myjeeva.digitalocean.common.ActionType;
+
+public class VolumeAction {
+
+	@Expose
+	private ActionType type;
+	
+	@Expose
+	@SerializedName("droplet_id")
+	private Integer dropletId;
+	
+	@Expose
+	@SerializedName("region")
+	private String regionSlug;
+	
+	@Expose
+	@SerializedName("volume_name")
+	private String volumeName;
+	
+	@Expose
+	@SerializedName("size_gigabytes")
+	private Integer sizeGigabytes;
```

Comment:
Please change datatype and variable name like [here](https://github.com/jeevatkm/digitalocean-api-java/blob/master/src/main/java/com/myjeeva/digitalocean/pojo/Image.java#L64). Thanks.